x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/RoleDescriptorTests.java

```diff
@@ -572,6 +619,8 @@ public void testParseIndicesPrivilegesSucceedsWhenExceptFieldsIsSubsetOfGrantedF
     }
 
     public void testParseIndicesPrivilegesFailsWhenExceptFieldsAreNotSubsetOfGrantedFields() {
+        resetFieldPermssionsCache();
```

Comment:
Just a question: Is this reset necessary or nice to have just in case?

Reply:
It's no longer necessary. It was needed in an earlier version where sometimes tests ran without a cache, because the behaviour with/without the cache was subtly different.
Now it's just there because it makes sense to have a known clean base for the test (and not have it be at risk of flakiness due to what's existing in the cache)